doc/tech/usecases.md

```diff
@@ -11,3 +11,26 @@
 
 <a name="5.1"></a>
 ## 5.1 Edge
+
+3-Approach & Scope  : 
+
+Edge Approach:
+-	 All gaps under Reference Model , Reference Architecture 01 -OpenStack- , Reference Architecture 02 - K8s will be identified and Edge requirements will be fulfilled  under Reference Model , Reference Architecture 01 -OpenStack- , Reference Architecture 02 - K8s  
+
+Edge scope under CNTT will cover :
+
+-	Define Edge locations based on use case 
+-	Define some guideline about external factors that can affect the edge like WAN latency based on telco use case 
+-	Define VNF/CNF profiles ( if needed )specific to Edge use case such as profile required acceleration technologies 
+-	Define resources requirements in terms of compute , Storage and networking , new concept can be introduced such as hyper coverage infrastructure 
```

Comment:
minor typos:

"Define resource requirements in terms of compute, storage and networking; new concepts, such as hyper converged infrastructure, can also be introduced"